samples/net/zperf/src/zperf_shell.c

```diff
@@ -391,36 +391,34 @@ static void shell_udp_upload_print_stats(const struct shell *shell,
 				      "LAST PACKET NOT RECEIVED!!!\n");
 		}
 
-		shell_fprintf(shell, SHELL_NORMAL,
-			      "Statistics:\t\tserver\t(client)\n");
-		shell_fprintf(shell, SHELL_NORMAL, "Duration:\t\t");
+		printf("Statistics:\t\tserver\t(client)\n");
+		printf("Duration:\t\t");
 		print_number(shell, results->time_in_us, TIME_US,
 			     TIME_US_UNIT);
-		shell_fprintf(shell, SHELL_NORMAL, "\t(");
```

Comment:
It is not recommended to modify original code in our repo temporary, since it will cause conflicts when we rebase to the next release.
Could you keep these changes in your local.

Reply:
OK. This shell_fprintf is not working upon unisoc base.
Anyway, I'm trying to get the below PR merged into zephyrpr master.
https://github.com/zephyrproject-rtos/zephyr/pull/13352

Unisoc PR:
https://github.com/unisoc/zephyr/pull/111

Reply:
> It is not recommended to modify original code in our repo for temporary, since it will cause conflicts when we rebase to the next release.
> Could you keep these changes in your local.

OK. This shell_fprintf is not working upon unisoc base.
Anyway, I'm trying to get the below PR merged into zephyrpr master.
[zephyrproject-rtos#13352](https://github.com/zephyrproject-rtos/zephyr/pull/13352)

Unisoc PR:
#111